### DISCUSSION TITLE
Possible Datasets

### DISCUSSION THREAD

**Discussion by valverde667** [pyDataSuite/.github#6] [Ideas]
- Dataset from Kaggle using computational fluid dynamics (CFD) in an aerodynamics dataset. https://www.kaggle.com/datasets/shtrausslearning/external-cfd-aero

**Comment by joeyame** [comment]
Was thinking this morning - might be easier if we spin up a homebrew solution. I'm starting a repo:
https://github.com/pyDataSuite/dataset-generator

where a dataset will be generated just from built-in computer sensors, like temperature, CPU load, RAM usage, etc. Should be easy to generate enough real-life sensor data.

**Reply by joeyame** [reply]
I'm gonna spend this afternoon working on this
